### PR TITLE
Fix/Improve API-Usage-STAC Endpoints Page

### PR DESCRIPTION
**Pull request by imanshafiei540**
Changes included here are only inside `API/APIUsage/Endpoints.md`:

[First Commit]

- Improved `Downloading Assets` section
- Improved/Fixed `Fields Extension` section
- Improved "Cloud Masks" section
- Some Markdown fixes

image

image

image

### CODE REVIEW

File: API/APIUsage/Endpoints.md [hunk 1]
```diff
@@ -204,18 +203,36 @@ The Fields Extension allows you to specify which fields are returned from the AP
...
+```bash
+curl --location 'https://api.earthdaily.com/platform/v1/stac/search' \
+--header 'Content-Type: application/json' \
+--data  '{
+  "fields": {
+    "include": ["id",  "properties.datetime",  "assets.aot"],
+    "exclude": ["links",  "geometry"]
+  }
+}'
+```
```
Comment (NeelamRG) [inline API/APIUsage/Endpoints.md]: @imanshafiei540 - If you click on command line link, you can see these examples. This seems to be a repetition to me

File: API/APIUsage/Endpoints.md [hunk 2]
```diff
@@ -238,25 +255,45 @@ GET https://api.earthdaily.com/platform/v1/stac/search?sortby=-eo:cloud_cover
...
+--data '{
+    "collections": ["sentinel-2-l2a"],
+    "query": {
+        "eda:ag_cloud_mask_available": {
+            "eq": true
+        }
+    }
+}'
+```
+<!-- space for examples -->
```
Comment (NeelamRG) [inline API/APIUsage/Endpoints.md:273]: This is also done in the command line example

File: API/APIUsage/Endpoints.md (hunk 1 above)
Comment (imanshafiei540) [reply API/APIUsage/Endpoints.md]: Thanks for the comments. To be honest, I'm not sure why we have GET examples here but `cURL` POST examples in the `Examples` section. I think there should be one place to include both GET and POST examples.
For the time being, I removed both code blocks.

Comment (NeelamRG) [reply API/APIUsage/Endpoints.md]: @imanshafiei540 - I think the main page is to give a templated URL (either GET or POST) but actual examples are examples with parameters. Let me know if you see differently anywhere

**Review by NeelamRG — APPROVED** [review]

**Pull request merged**